vulcanai/models/basenetwork.py

```diff
@@ -2,8 +2,6 @@
 """Defines the basenetwork class."""
 # Core imports
 import abc
-import torch
```

Comment:
you're removing these because they are imported on __init__ ?

Reply:
yes, and not direclty used in this class.